src/schema/v2/fields/money.ts

```diff
@@ -190,6 +190,7 @@ export const Money = new GraphQLObjectType<any, ResolverContext>({
     currencyPrefix: {
       type: GraphQLString,
       description: "The symbol used for the currency",
+      resolve: ({ currency }) => currencyPrefix(currency),
```

Comment:
@Mitch-Henson, note I had to resolve one more spot to make this work universally